src/OrderMapper.ts

```diff
@@ -60,6 +64,13 @@ export class OrderMapper {
             this.oasisV1SimpleExchangeWrapper.getAddress(),
         };
 
+      case OrderType.OasisV2:
```

Comment:
`OasisV2` does not exist on `OrderType`, you have to add it

How did this compile / your editor linter not catch it?